examples/basics/demo_transforms.py

```diff
@@ -121,7 +121,7 @@
 # Symmetrize function with respect to transform
 f_s = t.symmetrize(f, average=True)
 dinv.utils.plot(
-    [x, f(x), f_s(x)], titles=["Orig", "$f(x)$", "$\sum_i T_i^{-1}f(T_ix)$"]
+    [x, f(x), f_s(x)], titles=["Orig", "$f(x)$", "$\\sum_i T_i^{-1}f(T_ix)$"]
```

Comment:
Why is this needed? Seems like the Sigma looks fine on the [example](https://deepinv.github.io/deepinv/auto_examples/basics/demo_transforms.html)
(Secondly I still don't know why the example looks so ugly, any ideas @tomMoral ?)

Reply:
`\s` raises a warning when building the doc because it is not well encoded.
We can either use a raw string or escape the `\`. I choose to escape it but we could simply add `r` in from of the string.
As you prefer.

For the look of the example, I think the issue is that the text is too large, we should play with [`subpots_adjust`](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.subplots_adjust.html#matplotlib.pyplot.subplots_adjust) to make enough room for the equation, or reduce the font size to make it fit.